<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="showStatusLbl">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="currentWordLbl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Current word</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Wide Latin</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HangMan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
